index.html

```diff
@@ -213,19 +214,19 @@ <h2>LET’S CONNECT</h2>
                 </div>
               
                 <div class="container">
-  <form action="/action_page.php">
+  <form action="https://formspree.io/f/xeqdveok" method="post">
     <h2 class="first-text-contact">Contact me</h2>
     <p class="second-text-contact">If you have an application you are interested in developing,
        a feature that you need built or a project that needs coding. 
       I’d love to help with it </p>
 
     
-    <input type="text" id="fname" name="firstname" placeholder="Your name.." maxlength="30">
+    <input type="text" id="fname" name="firstname"  maxlength="30" placeholder="Your name.." required>
 
-    <input type="text" id="email" name="email" placeholder="eg.thszzc@gmail.com" maxlength="30">
+    <input type="text" id="email" name="email"  maxlength="30" placeholder="eg.thszzc@gmail.com" required>
```

Comment:
- [ ] Just as you added type text to the input above, there is also type email for email so that it would help you with validation. This is the idea of html 5 and I think you should take advantage of it.